site/guide/manage-users.qmd

```diff
@@ -35,11 +38,17 @@ To perform these next steps, you must be a group administrator or a superuser.
    
    Users now will have the option to accept the invitation through the email they receive.
 
-3. You can find pending invitations at the **Pending Invites** section.
+### Monitoring user invitations
+
+1. Monitor the status of invitations in the **Invites** section. Look for **Pending** status for users who haven’t accepted their invites yet.
```

Comment:
This is incorrect and refers to an earlier version of the UI, I believe. At a minimum, this is now the **Pending Invites** section but I'm fairly certain there is no longer a **Pending** status, either. 

See https://docs.validmind.ai/releases/2024-mar-27/highlights.html#user-management-improvements for some info I published in our release notes last month.

Reply:
The latest update no longer has these lines.

Reply:
Current docs are correct, thank you for making this update.